docs/grading_lists/single_list_grader.md

```diff
@@ -2,18 +2,26 @@
 
 If you want a response to be a delimiter-separated list of items, you can use a special ItemGrader called `SingleListGrader` to perform the grading. You need to specify a subgrader (which must be an ItemGrader, and could even be another SingleListGrader) to evaluate each individual item. The basic usage is as follows.
 
-```python
-grader = SingleListGrader(
-    answers=['cat', 'dog'],
-    subgrader=StringGrader()
-)
+```pycon
+>>> from mitxgraders import *
+>>> grader = SingleListGrader(
+...     answers=['cat', 'dog'],
+...     subgrader=StringGrader()
+... )
+>>> grader(None, 'cat, dog') == {'grade_decimal': 1.0, 'msg': '', 'ok': True}
+True
+>>> grader(None, 'dog, cat') == {'grade_decimal': 1.0, 'msg': '', 'ok': True}
+True
+>>> grader(None, 'cat, octopus') == {'grade_decimal': 0.5, 'msg': '', 'ok': 'partial'}
+True
+
 ```
 
 To receive full points for this problem, a student would enter `cat, dog` or `dog, cat` into the input box. Entering `cat, octopus` or just `cat` will receive half points. The `answers` key is a python list of the individual answers you wish to use.
 
 You can use a tuple of lists to specify multiple lists of answers, just like normal ItemGraders.
 
-```python
+```pycon
```

Comment:
I see change python -> pycon, but you still need the prompts `>>>` and `...` in order to make this run as a doctest. Also, I suspect the `pycon` syntax highlighting will look wrong without the prompts, but I am not sure.

(Oops, I guess I never submitted this comment in earlier review)

Reply:
I think I was ambitious and thought I'd update the entire doc, but never quite got around to it... Well, maybe I'll just do so now.